handbook/engineering/onboarding.md

```diff
@@ -31,6 +31,9 @@ As you are completing these tasks, you may notice documentation, processes, or c
 - Read through the rest of the engineering handbook to learn more about how we operate.
 - Setup your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/local_development.md#step-1-install-dependencies). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
 - Make yourself an admin on sourcegraph.com and sgdev.org by updating the database directly (this is not what a normal user would do, but doing it this way will expose you to useful knowledge). Relevant documentation:
+  - Hint: Make sure you have an account at these locations before editing the database
```

Comment:
What locations is this referring to? Is there a missing link here?

Reply:
`sourcegraph.com`, `sourcegraph.sgdev.org`, & `k8s.sgdev.org`
They are referenced on the lines below that point to deployments. 
https://about.sourcegraph.com/handbook/engineering/deployments
I think both these locations would benefit from being changed to links.